### ISSUE TITLE
Convert the US-China trade agreement

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/us-trade-agreements#1]
Since this is hot off the press we're going to use this as an example:

https://ustr.gov/sites/default/files/files/agreements/phase%20one%20agreement/Economic_And_Trade_Agreement_Between_The_United_States_And_China_Text.pdf

Thanks @manuel489 !

(in the meantime please encode it as MPFD for pargraph numbering)

**Comment by manuelfuenmayor** [comment]
Hi @ronaldtse 
I have added the converted file here: https://github.com/metanorma/us-trade-agreements 

Now, as this doctype is not supported yet by Metanorma, there are some important comments to make:
* The paragraph numbering is done manually. (Nick and I talked about this. Are you agree?)
* I put the word "Article" (along with its numbering) directly in the corresponding section titles.
* I didn't put any paragraph cross-reference in the document; it seems it is not supported yet.
* I put the words "Annex" and "Appendix" in the corresponding section titles because such words don't show up in the output.  And I decided to do it like this considering that there are several references in the content to these 'Annexes' and 'Appendixes'.

Ok, I think that's all I wanted to point out. Let me know if I should make corrections about this.

Thanks!
